MomsSpaghetti/Pages/Index.cshtml

```diff
@@ -83,194 +82,182 @@
+                            var first = propsArray[1].Value[0]["title"];
 
-                    }
+                        <h1>@Model.SearchRecipes.Result["recipes"][0]["title"]</h1>
+                        <h1>@Model.SearchRecipes.Result["recipes"][1]["title"]</h1>
+                                    *@
 
-                </ul>
+                                    foreach (var recipe in Model.SearchRecipes.Result["recipes"])
+                                    {
+                                        <li>
+                                            <a class="results__link results__link--active" href="?recipeId=@recipe["recipe_id"]">
+                                                <figure class="results__fig">
+                                                    <img src="@recipe["image_url"]" alt="Recipe">
+                                                </figure>
+                                                <div class="results__data">
+                                                    <h4 class="results__name">@recipe["title"]</h4>
+                                                    <p class="results__author">@recipe["author"]</p>
+                                                </div>
+                                            </a>
+                                        </li>
+                                    }
 
 
-                <div class="results__pages">
-                    <!--
-                    <button class="btn-inline results__btn--prev">
-                        <svg class="search__icon">
-                            <use href="img/icons.svg#icon-triangle-left"></use>
-                        </svg>
-                        <span>Page 1</span>
-                    </button>
-                    <button class="btn-inline results__btn--next">
-                        <span>Page 3</span>
-                        <svg class="search__icon">
-                            <use href="img/icons.svg#icon-triangle-right"></use>
-                        </svg>
-                    </button>
-                    -->
-                </div>
-            </div>
+                                }
 
-            <div class="recipe">
-                @if (Model.Recipe.Title != null)
-                {
 
+                            }
 
+                        </ul>
 
 
+                        <div class="results__pages">
+                            <!--
+            <button class="btn-inline results__btn--prev">
+                <svg class="search__icon">
+                    <use href="img/icons.svg#icon-triangle-left"></use>
+                </svg>
+                <span>Page 1</span>
+            </button>
+            <button class="btn-inline results__btn--next">
+                <span>Page 3</span>
+                <svg class="search__icon">
+                    <use href="img/icons.svg#icon-triangle-right"></use>
+                </svg>
+            </button>
+            -->
```

Comment:
Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.